src/bootstrap/step.rs

```diff
@@ -789,6 +786,10 @@ pub fn build_rules<'a>(build: &'a Build) -> Rules {
          .dep(move |s| s.name("tool-build-manifest").target(&build.config.build).stage(0))
          .run(move |_| dist::hash_and_sign(build));
 
+    rules.install("install", "path/to/nowhere")
+         .dep(|s| s.name("default:dist"))
```

Comment:
Right.

Since the rls README recommands adding rls, rust-analysis *and* rust-src to rustup, I figured rust-src was useful for it to work properly, so I didn't want to drop that part entirely. That's why I kept it as is and just put the actual full fist tarball creation (with the call to cargo vendor and so forth) in a conditional.